src/views/menus/AccountMenu.tsx

```diff
@@ -87,6 +95,11 @@ export const AccountMenu = () => {
 
   const { appleAppStoreUrl, googlePlayStoreUrl } = useMobileAppUrl();
 
+  const showConfirmPendingDeposit =
+    walletType === WalletType.Keplr &&
+    MustBigNumber(usdcBalance).toNumber() > AMOUNT_RESERVED_FOR_GAS_USDC &&
```

Comment:
Instead of using `toNumber()` I think you can use `MustBigNumber(usdcBalance).gt(AMOUNT_RESERVED_FOR_GAS_USDC)`

Reply:
since usdcBalance is used in the below comparison create `cosnt usdcBalanceBn = MustBigNumber(usdcBalance)`